app/controllers/members_controller.rb

```diff
@@ -8,5 +8,11 @@ def history
 
   def loans
     @loans = current_member.loans.includes(:item).order(:due_at)
+    @holds = current_member.holds.includes(:item)
+  end
+
+  def delete_hold
```

Comment:
Eventually we might want to move this to a `Member::HoldsController#destroy` action, but until we get there, this works.

Reply:
Agreed, I almost did it but it'd conflict with the other team's work.